src/App.js

```diff
@@ -1,6 +1,7 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
+import 'typeface-roboto'; // eslint-disable-line
```

Comment:
I think this import statement should be in line 1, that's a nicer way to see that there's an import with side effects.
What is the error message of the linter?

Reply:
Its an eslint error: 

``` Missing file extension "css" for "typeface-roboto"  import/extensions```